debian/base/include/dub

```diff
@@ -205,11 +252,15 @@ if __name__ == '__main__':
     check_env = actions.add_parser('ensure', description='Check if env var exists.')
     check_env.add_argument('name', help='Name of env var.')
 
-    check_env = actions.add_parser('wait', description='wait for network service to appear.')
+    check_env = actions.add_parser('wait', description='Wait for network service to appear.')
     check_env.add_argument('host', help='Host.')
-    check_env.add_argument('port', help='Host.', type=int)
+    check_env.add_argument('port', help='Port.', type=int)
     check_env.add_argument('timeout', help='timeout in secs.', type=float)
 
+    check_env = actions.add_parser('url-ready', description='Wait for a URL to be retrievable.')
```

Comment:
should we call this `http-ready`?  That's what it is.  We're not handling any other types of URLs